pkg/workload/workload.go

```diff
@@ -1114,8 +1123,7 @@ func ReportEvictedWorkload(recorder record.EventRecorder, wl *kueue.Workload, cq
 }
 
 func ReportPreemption(preemptingCqName kueue.ClusterQueueReference, preemptingReason string, targetCqName kueue.ClusterQueueReference, wl *kueue.Workload) {
-	durationToPreemption := workloadsWithPodsReadyToEvictedTime(wl)
```

Comment:
Why we can drop the preemption duration?

Reply:
Because `PodsReadyToEvictedTime` is already accounted via workload.ReportEvictedWorkload.

However, this function was not called during preemption, now I call it from applyPreemptionWithSSA.

See https://github.com/kubernetes-sigs/kueue/pull/6420 for the ease of reviewing I extracted the change